src/main/java/gui/modal/CameraShotCreationModalView.java

```diff
@@ -75,11 +73,11 @@
 
     // No touching these constants. They work well for all general cases,
     // and there is no reason to change them ever again.
-    private final int generalSize = 10000;
-    private final int generalSpacing = 10;
-    private final int generalPadding = 20;
-    private final int textAreaMinWidth = 350;
-    private final int cameraAreaMinWidth = 250;
+    private static final int generalSize = 10000;
+    private static final int generalSpacing = 10;
```

Comment:
constants should be IN_CAPITAL_LETTERS_ONLY. can you change this? Should be easy with the refactor function of the ide